src/test/java/com/jcabi/http/request/FakeRequestTest.java

```diff
@@ -49,6 +49,21 @@
 public final class FakeRequestTest {
 
     /**
+     * Reason phrase.
+     */
+    private static final String REASON_OK = "OK";
+
+    /**
+     * Body message: how are you?
+     */
+    private static final String BODY_HOW_ARE_YOU = "how are you?";
+
+    /**
+     * Path of hello all request.
+     */
+    private static final String PATH_HELLO_ALL = "/helloall";
```

Comment:
@antonini I think we should not coupling tests together, we can put any different string here, right?